### PR TITLE
tracing: Add an example showing how to set up Jaeger-native tracing.

### PR DESCRIPTION
**Pull request by rnburn**
Signed-off-by: Ryan Burn <ryan.burn@gmail.com>

*Description*:
Demonstrates how to set up tracing with a native Jaeger client.

*Risk Level*: Low 

*Testing*: Ran the example locally on my machine.

*Docs Changes*:
Added a doc to describe the sandbox.

### CODE REVIEW

**Comment by rnburn** [comment]
cc @isaachier - if you want to review the Jaeger setup and docs.

**Comment by htuch** [comment]
Merge master to fix build with https://github.com/envoyproxy/envoy/pull/3317.

File: docs/root/start/sandboxes/jaeger_native_tracing.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.. _install_sandboxes_jaeger_native_tracing:
+
+Jaeger Tracing
```
Comment (mattklein123) [inline docs/root/start/sandboxes/jaeger_native_tracing.rst]: Can you look at the rendered docs? There are a few oddities:
1) We show "Jaeger tracing" twice. Can you change the titles to make it more clear what they are about?
2) Some of the links are not properly linkified (I think missing '_' at the end of external links). This I think is a problem in both the old Jaeger stuff and the new text.

Comment (rnburn) [reply docs/root/start/sandboxes/jaeger_native_tracing.rst]: Should be fixed now. Thanks.

**Pull request merged**